data-prepper-plugins/s3-sink/src/test/java/org/opensearch/dataprepper/plugins/sink/S3SinkServiceTest.java

```diff
@@ -263,7 +260,7 @@ void test_output_with_uploadedToS3_success_records_byte_count() throws IOExcepti
 
     @Test
     void test_output_with_uploadedToS3_failed() throws IOException {
-        when(s3SinkConfig.getBucketOptions().getBucketName()).thenReturn(null);
+        when(s3SinkConfig.getBucketName()).thenReturn(null);
```

Comment:
Can we use a random string here? We should be using realistic values. Same is true for `test_retryFlushToS3_negative`